local-interchaintest/src/tests.rs

```diff
@@ -0,0 +1,80 @@
+use super::util;
+use serde_json::Value;
+use std::error::Error;
+
+const ERROR_MARGIN_PROFIT: u64 = 100000;
+
+pub fn test_profitable_arb(arbfile: Value) -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    let arbs = arbfile.as_array().expect("no arbs in arbfile");
+
+    util::assert_err("!arbs.is_empty()", arbs.is_empty(), false)?;
```

Comment:
passing the `"!arbs.is_empty()"` is confusing, stick to regular messages. it'd get picked up by grep-like functions and can cause confusion.

Reply:
Minor nit, don't think this is necessary to change.